CPM/Collaborative-Mark-Policy-draft.md

```diff
@@ -120,10 +120,18 @@ You may use the OSM marks on your own website to identify a hyperlink to the OSM
 You may use the OSM wordmarks as necessary to comply with the attribution requirements imposed by the OSM project's database and copyright licensing terms. For more information on OSM's database and copyright licence, and how to credit us, see [the OSM website](https://www.openstreetmap.org/copyright).
 
 ### 3.4. Make your own branded stuff
-You may create things with the marks for your own use. These can be t-shirts, caps, desktop wallpapers, and even cakes! But please don’t sell them. 
+You may create things with the marks for your own use. These can be t-shirts, caps, desktop wallpapers, and even cakes! You may also give these items to other people, either for free or for money, as long as you don't do this professionally or ask for more than cost recovery. (Printing a batch of t-shirts for an event is ok; setting up a web shop where people can order t-shirts is not.)
+
+If you want to sell stuff with OSM's brand on it professionally, you may request a licence under Section 4.4. 
```

Comment:
@simonpoole I'm not sure what difference you intended between this line and the next one?